event-sourcing/event-repository/event-repository-jdbc/src/main/java/uk/gov/justice/services/eventsourcing/repository/jdbc/eventlog/EventLogJdbcRepository.java

```diff
@@ -63,19 +62,16 @@ public void insert(final EventLog eventLog) throws InvalidSequenceIdException {
             throw new InvalidSequenceIdException(format("Version is null for stream %s", eventLog.getStreamId()));
         }
 
-        try (Connection connection = getDataSource().getConnection();
-             PreparedStatement ps = connection.prepareStatement(SQL_INSERT_EVENT_LOG)) {
-
+        try (final PreparedStatementWrapper ps = preparedStatementWrapperOf(SQL_INSERT_EVENT_LOG)) {
             ps.setObject(1, eventLog.getId());
             ps.setObject(2, eventLog.getStreamId());
             ps.setLong(3, eventLog.getSequenceId());
             ps.setString(4, eventLog.getName());
             ps.setString(5, eventLog.getMetadata());
             ps.setString(6, eventLog.getPayload());
-//            ps.setTimestamp(7, toSqlTimestamp(eventLog.getDateCreated()));
```

Comment:
This will need to be re-enabled when the date_created can be added as a non-breaking change. But will leave it up to you if we want it removed or just commented out.